javalib/src/main/scala/java/lang/Double.scala

```diff
@@ -211,7 +211,7 @@ object Double {
     }
 
   @inline def doubleToLongBits(value: scala.Double): scala.Long =
-    if (value != value) 0x7ff8000000000000L
+    if (value != value) 0x7FF8000000000000L
```

Comment:
Is this configurable? Can we ask scalafmt to preserve lowercase for hexadecimal digits?

Reply:
These are the defaults for just the character at the end as far as I know. It looks like they just uppercase or lowercase the whole number rather than just the last character.
```
literals.long = Upper
literals.float = Lower
literals.double = Lower
```
I also looked checked and if the hexidecimal number is not a `long` it leaves those alone so in `java.lang.Character` some hexadecimals are lowercase and some are uppercase.

I think I should just roll back that change and report it to `scalafmt` until a change can be made. 

The last character casing above makes sense but not changing the hex portion unless they are changed everywhere. Also, if you have E Notation like `10E-3f` it changes to `10e-3f` and if you have `10e-3L` it would change to `10E-3L`. This is clearly weird.

Reply:
The whole idea is you have common decent standards and then forget about it as the tool does it for you.